app/views/ingredients/edit.html.erb

```diff
@@ -20,7 +20,7 @@
       <div id="navbarMenu" class="navbar-menu">
         <div class="navbar-end">
 
-          <a href="#" class="navbar-item">
+          <a href="/inventory" class="navbar-item">
```

Comment:
Same here!
I'm not leaving a comment for each one 😅

Reply:
Yep this in mind all around you application